.zenodo.json

```diff
@@ -327,6 +327,7 @@
     },
     {
       "name": "Yaniv, Ziv"
```

Comment:
Please add a `,` for valid JSON.

Reply:
Fixed and force pushed. Can't seem to add one line without making a mistake ;).